Adding Apartment data model

tamirho opened the pull request:
- Adding Apartment class that connects to the User as a foreign key
- Adding City class to create dynamic choices for the 'Apartment' city field.
- Alter the users/admin.py to view Apartment and city classes.

Noice: I put the 'img' field in the Apartment model in a comment because I didn't want to prevent the rest from moving forward.
I prefer to research a little more about the Django image field and add it as a future feature

fix: #33 

<img width="530" alt="Apartment" src="https://user-images.githubusercontent.com/65215909/113590380-3ddc0b80-963b-11eb-9efe-74492d7fbe1d.PNG">

DanielMy777 reviewed (commented):
Looking good!!
(other than '.idea' that we are awaiting an answer for)

simzacks commented on users/models.py:
```diff
@@ -68,3 +69,34 @@ class User(AbstractBaseUser, PermissionsMixin):
+
+
+class Owner(models.Model):
+    user = models.OneToOneField(AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
```
Are you planning on adding additional fields to Owner? If not, this model has no purpose.

simzacks replied:
I'm just waiting for an answer to this question before approving.

tamirho replied:
According to yesterday's session, I understand from the group that we are planning on adding additional fields to Owner.
But now that I reconsider it, I think it might really be better to give up the Owner class.
I would like to hear more from the other teammates before I take action and delete it.
@Micha546 @AmitAharoni @DanielMy777 @nadavsu

DanielMy777 replied:
Lets remove this model, I don't see a reason to keep it other than organization.

Micha546 replied:
I also think that we should remove the Owner class and only use Apartment.
We don't really have a use for the Owner right now...

AmitAharoni replied:
Just keep the Aprtment class, It will be enough

tamirho replied:
Ok, so I will remove the Owner and change the Apartment 'owner' field.

simzacks commented on users/models.py:
```diff
@@ -68,3 +69,34 @@ class User(AbstractBaseUser, PermissionsMixin):
+    numOfRooms = models.IntegerField()
+    startDate = models.DateField()
+    content = models.TextField(blank=True)
+    isRelevant = models.BooleanField(default=True, blank=True)
```
What is the purpose of the isRelevant field? Is this for the owner to mark that it shouldn't be shown, or for some other purpose?

tamirho replied:
Yes, I thought that if the owner found a partner he could mark that the apartment is no longer relevant and then hide it from the search but keep it in history.
In addition, if after a certain time he is looking for another partner with the same apartment, he can return it from history

simzacks reviewed (commented):
I think the .idea folder is the settings for your IDE. That shouldn't generally be included in version control. To remove it, you would use  `git rm`  and then add it to the .gitignore

tamirhouri committed:
Adding Apartment class

- Adding Apartment class that connect to User as a foreign key
- Adding City class to create dynamic choices for 'Apartment' city
  field.

tamirhouri committed:
Alter the users/admin.py

-Alter the users/admin.py to view Apartment and city classes

Omeramsc reviewed (approved):
LGTM
